storage/iscsi/params/runtest.sh

```diff
@@ -38,12 +36,13 @@ function cleanup
 function runtest
 {
     typeset fwroot=$(stqe_get_fwroot)
-    pushd "." && rlRun "cd $fwroot"
-    rlRun "stqe-test run -t iscsi/iscsi_params.py"
-    popd
+    cki_cd $fwroot
+    cki_run_cmd_pos "stqe-test run -t iscsi/iscsi_params.py" || \
```

Comment:
So, if this fails, the directory will be wrong because the `cki_pd` will not be run? Shouldn't that be avoided?

Reply:
Good catch, updated. Please refer to:
* http://pastebin.test.redhat.com/799524